content/blog/pulumi-in-a-cloud-native-world/index.md

```diff
@@ -0,0 +1,500 @@
+cluster, we opted here for the [vCluster](https://www.vcluster.com/) project. The vCluster project allows us to create a
+Kubernetes cluster on demand, which runs as containers on the control plane cluster. The moment the cluster is created,
+we add it to the Argo CD as spoke cluster. This way, we can roll out applications to the development cluster in the same
+way as we would do for production clusters.
+
+See the definition of the `ClusterPolicy` object in the [Kyverno](#bonus-kyverno) section.
+
+#### Ordering a New Production Kubernetes Cluster
+
+<img src="order_cluster.png" width="500px">
+
+We use Backstage to order a new production Kubernetes cluster. This time, we use the Pulumi code to create the cluster
+in the cloud provider of choice. After the creation of the cluster, we add it to the Argo CD as a production cluster.
+This way, we can roll out applications to the production cluster in the same way as we would do for development
+clusters.
+
+#### Ordering Other Infrastructure
+
+<img src="order_rest.png" width="500px">
+
+We use Backstage to order other infrastructure components, such as databases, message queues, and storage. We use again
+the Pulumi Kubernetes Operator to create the infrastructure components in the Kubernetes cluster. This way, we can
+manage all the infrastructure components in the same way as we manage the Kubernetes clusters.
+
+## Conclusion and Pulumi Cloud as an Off-the-Shelf Solution
+
+A lot of organizations are building their internal developer platforms using CNCF projects to fulfill the needs of their
+internal customers. The `KEBAP` stack provides a structured approach to integrate these technologies into an effective
+platform engineering solution. We also saw that Pulumi plays a crucial role in this stack by taking care of the
+management of the cloud infrastructure.
+
+This `KEBAP` stack offers several benefits, including:
+
+1. A secure, automated, and auditable environment
+2. Reproducible and programmable infrastructure-as-code
+3. Unified management of cloud resources, Kubernetes, and application deployments
+4. Faster onboarding for new team members
+5. Easier compliance with regulatory requirements due to increased auditability
+
+But there is also a downside: The `KEBAP` stack requires a significant investment in time and resources to set up and
+maintain as you are in full charge of the integration of the components as well as operating them. All the projects
+under the umbrella of CNCF are not automatically implying that they are compatible with each other out of the box. Any
+platform engineering team will need to invest continuously time and resources to keep the stack up-to-date and running.
+This is a not negligible investment to consider for any organization.
+
+For this reason, we at Pulumi offer [Pulumi Cloud ](/product/pulumi-cloud). A fully managed Infrastructure as Code
```

Comment:
As I said above, I think KEBAP stack doesn't preclude the usage of Pulumi Cloud. I would rather say 

For this reason, we have another reference architecture that uses components from all of the Pulumi Platform for an end to end fully-managed solution. This solution takes care of the integration of the different components of and internal developer portal and provides a carefree experience of managing your infrastructure at scale. Stay tuned...

Reply:
Like, and good handover to the following block posts!